### ISSUE TITLE
[v4] | BottomSheetModal content not accessible by screen reader on iOS

### ISSUE THREAD

**Issue by gracjandotdev** [gorhom/react-native-bottom-sheet#620]
# Bug

Contents of the BottomSheetModal component (also BottomSheet) are not accessible by screen reader on iOS.

## Environment info

| Library                         | Version |
| ------------------------------- | ------- |
| @gorhom/bottom-sheet            | 4.0.2   |
| react-native                    | 0.63.3   |
| react-native-reanimated         | 2.2.0  |
| react-native-gesture-handler    | 1.10.2   |

## Steps To Reproduce

1. Turn on voice over in iOS accessibility settings 
2. Navigate to the application
2. Open Bottom Sheet Modal
3. Try to tap on elements inside. It will focus only the wrapper, but not contents

Describe what you expected to happen:

1. Expect screen reader to focus and announce items contained within a Bottom Sheet.

## Reproducible sample code

https://snack.expo.dev/@jangsoori/ashamed-kiwi

**Comment by kkureli** [comment]
Hi, any solution?
